aea/cli/utils/click_utils.py

```diff
@@ -160,6 +207,41 @@ def wrapper(f: Callable) -> Callable:
     return wrapper
 
 
+def registry_flag_() -> Callable:
```

Comment:
This will replace the `registry_flag` decorator but it's not fully integrated yet, Once I'm done updating all the necessary command I'll fix this